__tests__/app.test.js

```diff
@@ -43,26 +43,36 @@ describe("/api/topics", () => {
   });
 });
 
-describe("/api/topics", () => {
-    test("GET 200 - should receive an array of topic objects contains slug and description keys with strings as values", () => {
+describe("/api/articles/:article_id", () => {
+    test("GET 200 - should receive an article object with the correct article_id", () => {
       return request(app)
-        .get("/api/topics")
+        .get("/api/articles/4")
         .expect(200)
         .then((result) => {
-          expect(Array.isArray(result.body.topics)).toBe(true);
-          expect(result.body.topics.length).toBe(3);
-          result.body.topics.forEach((topic)=>{
-            expect(typeof topic.slug).toBe('string');
-            expect(typeof topic.description).toBe('string');
-          })
+          expect(typeof result.body.article.article_id).toBe('number');
+          expect(typeof result.body.article.title).toBe('string');
+          expect(typeof result.body.article.topic).toBe('string');
+          expect(typeof result.body.article.author).toBe('string');
+          expect(typeof result.body.article.body).toBe('string');
+          expect(typeof result.body.article.created_at).toBe('string');
+          expect(typeof result.body.article.votes).toBe('number');
+          expect(typeof result.body.article.article_img_url).toBe('string');
```

Comment:
.toHaveProperty or .toMatchObject could be good matchers to use here rather than just checking the type of the value